python/tvm/testing/plugin.py

```diff
@@ -0,0 +1,82 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Pytest plugin for using tvm testing extensions.
+
+TVM provides utilities for testing across all supported targets, and
+to more easily parametrize across many inputs.  For more information
+on usage of these features, see documentation in the tvm.testing
+module.
+
+These are enabled by default in all pytests provided by tvm, but may
+be useful externally for one-off testing.  To enable, add the
+following line to the test script, or to the conftest.py in the same
+directory as the test scripts.
+
+     pytest_plugins = ['tvm.testing.plugin']
+
+"""
+
+import pytest
+
+import tvm.testing.testing
+
+
+def pytest_configure(config):
+    """Runs at pytest configure time, defines marks to be used later."""
+    markers = {
```

Comment:
should this be a global?

Reply:
We could change it, but I don't know if there would be an advantage to doing so.  The main reason I would see for defining the markers in a global is so that they could be modified by other files being loaded.  The call to `pytest_configure` occurs prior to reading the test files being run, so the test files themselves can't change a global variable.  The only thing that could is a subdirectory's `conftest.py`, but those could just as easily define their own `pytest_configure` hook.

Reply:
my question was more a stylistic one--this looks like a constant

Reply:
Sorry, I'm not quite following the suggestion.  That is correct that it is a constant.  Is the suggestion to change it from lower-case `markers` to an upper-case `MARKERS` to indicate that this is a constant value?

Reply:
well, if it were a constant we could move it out of the function; then it would make sense to use capitalized `MARKERS =`. however, i see your point. it's fine to leave as is